### PR TITLE
Make the mongo seed files compatible with both BE

### PR DESCRIPTION
**Pull request by fpotier**
We should take into account all [these changes](https://github.com/SciCatProject/scicat-backend-next/tree/master/migrations)

### CODE REVIEW

File: services/backend/services/v4/compose.base.yaml [hunk 1]
```diff
@@ -7,6 +7,12 @@ services:
         condition: service_started
     volumes:
       - ./config/functionalAccounts.json:/home/node/app/functionalAccounts.json
+      - ../../../../entrypoints/loop_entrypoints.sh:/usr/local/bin/loop_entrypoints.sh
+      - ./entrypoints/db_migration.sh:/docker-entrypoints/0.sh
+    entrypoint:
+      - loop_entrypoints.sh
+      - docker-entrypoint.sh
```
Comment (fpotier) [inline services/backend/services/v4/compose.base.yaml:14]: I think this is useless but I did the same way it's done in v3

Comment (minottic) [reply services/backend/services/v4/compose.base.yaml:14]: yes, I think as well. I just wanted to be safer to always apply the latest default `docker-entrypoint` from the container, in case they will do something more useful in the future (for now it just allows to avoid explicitly writing `node` in the command whenever you just want to add options

File: services/backend/services/v4/compose.base.yaml [hunk 2]
```diff
@@ -7,6 +7,12 @@ services:
         condition: service_started
     volumes:
       - ./config/functionalAccounts.json:/home/node/app/functionalAccounts.json
+      - ../../../../entrypoints/loop_entrypoints.sh:/usr/local/bin/loop_entrypoints.sh
```
Comment (minottic) [inline services/backend/services/v4/compose.base.yaml:10]: could you please use `${PWD}` instead of ../../../? I think it improves readability

File: services/backend/services/v4/entrypoints/db_migration.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo "MONGODB_URI=$MONGODB_URI" >.env
```
Comment (minottic) [inline services/backend/services/v4/entrypoints/db_migration.sh:3]: why is this needed?

**Review by minottic — COMMENTED** [review]
very minor comments, looks good!

File: services/backend/services/v4/entrypoints/db_migration.sh (hunk 3 above)
Comment (fpotier) [reply services/backend/services/v4/entrypoints/db_migration.sh:3]: because the migration script uses the `dotenv` npm package which reads env variables from a `.env` file

**Review by minottic — APPROVED** [review]

File: services/backend/services/v4/compose.base.yaml (hunk 1 above)
Comment (consolethinks) [reply services/backend/services/v4/compose.base.yaml:14]: How do multiple entrypoints work?

File: services/backend/services/v4/entrypoints/db_migration.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
```
[the remark this answers is not in the document]
Comment (fpotier) [reply services/backend/services/v4/entrypoints/db_migration.sh:3]: container's workdir which I believe is `/home/node/app`

File: services/backend/services/v4/compose.base.yaml (hunk 1 above)
Comment (minottic) [reply services/backend/services/v4/compose.base.yaml:14]: they are executed one after the other, so, it would be `. loop_entrypoints.sh docker-entrypoint.sh`. One needs to implement the entrypoint then so that they can take another sh script as input. That's done with the exec "$@" bit which both entrypoints have
